src/components/twilio-voice-dialer/public/index.js

```diff
@@ -4,4 +4,16 @@
 
   const twilioVoiceDialer = document.querySelector('twilio-voice-dialer');
   twilioVoiceDialer.setToken(data.token);
+  twilioVoiceDialer.addEventListener('onTokenWillExpire', async (e) => {
+    const device = e.detail.device;
+    console.log('onTokenWillExpire', device);
+
+    const updateTokenResponse = await fetch(`/twilio-voice-dialer/token`);
+    const updateTokenData = await updateTokenResponse.json();
+    device.updateToken(updateTokenData.token);
+  });
+  twilioVoiceDialer.addEventListener('onIncoming', (e) => {
+    const call = e.detail.call;
+    console.log('onIncoming', call);
```

Comment:
Not sure the ideal example to put here, any suggestions?

Reply:
Maybe just a comment saying something like "you can handle call events here" or even handling a call event like `call.on('disconnect', ...)`.

Reply:
Is this component supposed to have full incoming and outgoing call functionality, or just outgoing?

Reply:
The component has both functionality. However we only expose the incoming event. See IDL: https://docs.google.com/document/d/1rHlTs9y7TcNZLaHx2aHJAgo7tEwqyvn-Izbf0vkoLYQ/edit#heading=h.xf1ox0g8pcv6